setup.py

```diff
@@ -82,15 +82,15 @@
     'invenio-access>=1.0.0a11',
     'invenio-accounts>=1.0.0b1',
     'invenio-accounts-rest>=1.0.0a1',
-    'invenio-assets>=1.0.0b4',
+    'invenio-assets>=1.0.0b6',
     'invenio-base>=1.0.0a14',
     'invenio-celery>=1.0.0b1',
     'invenio-collections>=1.0.0a3',
     'invenio-config>=1.0.0b1',
     'invenio-db[postgresql,versioning]>=1.0.0b3',
     'invenio-files-rest>=1.0.0a14',
-    'invenio-deposit>=1.0.0a7',
     'invenio-indexer>=1.0.0a9',
+    #'invenio-deposit>=1.0.0a6',
```

Comment:
let's hope for quick integration of https://github.com/inveniosoftware/invenio-deposit/pull/162